src/IonDecimal.ts

```diff
@@ -49,27 +49,68 @@ export class Decimal {
     public static readonly ONE = new Decimal(1, 0);
 
     /**
-     * Creates a new Decimal value.
-     * @param coefficient   See the {@link Decimal} class-level documentation for details.
-     * @param exponent      See the {@link Decimal} class-level documentation for details.
+     * Creates a new Decimal value. Three parameter sets are supported:
+     *
+     *  (text: string)
+     *      The provided text will be parsed into a Decimal value.
+     *
+     *  (coefficient: number, coefficient: number)
+     *      A Decimal value will be constructed using the provided coefficient and exponent values.
+     *
+     *  (coefficient: BigInt, coefficient: number, isNegative?: boolean)
+     *      A Decimal value will be constructed using the provided coefficient and exponent values.
+     *      Because the BigInt data type cannot represent -0, a third parameter called `isNegative` may be supplied
+     *      to explicitly set the sign of the Decimal following construction. If this flag is not specified,
+     *      the provided coefficient's sign will be used.
+     */
+    constructor(coefficient: number | JSBI | string,
+                exponent?: number,
+                isNegative?: boolean) {
+        if(typeof coefficient === "string") {
+            return Decimal.parse(coefficient);
+        }
+
+        if (!_hasValue(exponent)) {
+            throw new Error("Decimal's constructor was called with a numeric coefficient but no exponent.");
+        }
+
+        if (typeof coefficient === "number") {
+            return Decimal._fromNumberCoefficient(coefficient, exponent);
+        }
+
+        if(coefficient instanceof JSBI) {
+            if (!_hasValue(isNegative)) {
+                // If isNegative was not specified, infer isNegative from the coefficient.
+                // This will work for all values except -0.
+                isNegative = JsbiSupport.isNegative(coefficient);
+            } else if (isNegative && !JsbiSupport.isNegative(coefficient)) {
+                // If isNegative was specified, make sure that the coefficent's sign agrees with it.
+                coefficient = JSBI.unaryMinus(coefficient);
+            }
+            return Decimal._fromBigIntCoefficient(isNegative, coefficient, exponent);
```

Comment:
What about `! isNegative` and `JsbiSupport.isNegative(coefficient)`?  Is this a valid state?